projects/laji/src/app/shared-modules/spreadsheet/service/mapping.service.ts

```diff
@@ -428,7 +428,10 @@ export class MappingService {
         const ykjParts = value.split(':');
         if (ykjParts[0].length === ykjParts[1].length) {
           try {
-            return this.coordinateService.convertYkjToGeoJsonFeature(ykjParts[0], ykjParts[1]).geometry;
+            const data = latLngGridToGeoJSON([ykjParts[0], ykjParts[1]]);
+            if (data.geometry) {
+              value = data.geometry;
+            }
```

Comment:
Should we just return the `data.geometry` here instead of binding it to the `value` variable? Now the execution will continue to trying to `convertAnyToWGS84GeoJSON` (13 lines below), and if that fails, `null` will be returned. There could be cases when data that is valid for `latLngGridToGeoJSON` but invalid for `convertANyToWGS84GeoJSON`, and the value from `latLngGridToGeoJSON` would be lost.